paper/writeup.tex

```diff
@@ -662,45 +675,51 @@ \subsection{Dissimilarity between ARGs}
 Concretely, our method can be applied to any two tree sequences,
  as in the case of \figref{fig:conceptual_discrepancy},
  where we consider the pair of trees which exist at that break point.
-However, suppose that the two tree sequences have the same set of breakpoints
+However, to simplify notation suppose that the two tree sequences have the same set of breakpoints
 between trees,
 so that $T_1^{(1)}, \ldots, T_N^{(1)}$ are the trees in $\T_1$
 and $T_1^{(2)}, \ldots, T_N^{(2)}$ are the trees in $\T_2$.
 For a node $n$ and tree $T$
 let $S(T, n)$ denote the set of samples that inherit from $n$ in $T$,
 and for a pair of nodes $n_1$ and $n_2$ with $n_1$ in $\T_1$ and $n_2$ in $\T_2$,
 define
+% NSP-22NOV: The notation here is hard to read when subscripted. IMO it'd be better to use bracket notation here for the indicator function,
+% $\mathbb{I}[ S(T_k^{(1)}, n_1) = S(T_k^{(2)}, n_2) ]$
+% and also in next equation for consistency.
+% HF24-NOV: I think I disagree but we can get a third opinion.
```

Comment:
Either way is okay by me, but a third option is to define in the text like 
\mathcal{M}(T1, n1; T2, n2) = \{ k : S(T^{(1)}_k,\ n_1) = S(T^{(2)}_k,\ n_2) \}
and then have the sum over this set.